cron_script.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+cd $(dirname $0)
+
+. deleteENV/bin/activate
+
+# generate lists
+./phase0_generateuserlist_alt.py
+
+# notify users next to expire
+./phase0b_notify_users.py
+
+# change type of expired users
+./phase0c_change_category.py
+
+# stop servers of expired users
+if [ ! -f users_to_delete_phase3.txt ] ; then
+./phase1_generate_trust_ids.py
+./phase2_stopvms.py
+rm users_trusted_ids.txt
+fi
+
+# delete resources of users who expired days ago (i.e. accounts that were
+# processed by the phase2_stopvms some days ago)
+if [ -f users_to_delete_phase3.txt ] ; then
```

Comment:
this is not the else operation of the previous condition?

But if you want to check it again due to it is possible that there is Nothing to delete, then you should give a (log) message in order to inform that there is no users to delete in phase3 or there was an uncontrolled error.